packages/lib-auth/src/helpers/validate-config.js

```diff
@@ -0,0 +1,34 @@
+const yup = require('yup')
+
+const clientAppID = yup.string()
+  .matches(/^[a-z0-9]+$/i)
+  .required()
+
+const cookieName = yup.string()
+  .default('_Panoptes_session')
+
+const hostUrl = yup.string()
+  .url()
```

Comment:
`localhost` is currently rejected, but should be valid as well

Reply:
`localhost` should definitely not work.

Reply:
I'd modified my `cors-config` in my local instance of Panoptes - you're right, it shouldn't work.